src/Model/Settings.cs

```diff
@@ -174,29 +172,23 @@ public void Load(string filename)
                         {
                             if (r.NodeType == XmlNodeType.Element)
                             {
-                                string name = r.Name;
-                                object o = map[name];
-                                if (o != null)
+                                var name = r.Name;
+                                var entity = _map[name];
+                                if (entity != null)
                                 {
                                     object value = null;
-                                    if (o is Hashtable)
-                                    {
-                                        ReadHashTable(r, (Hashtable)o);
-                                    }
-                                    else if (o is Array)
-                                    {
-                                        value = ReadArray(name, (Array)o, r);
-                                    }
-                                    else if (o is IXmlSerializable)
-                                    {
-                                        IXmlSerializable xs = (IXmlSerializable)o;
+                                    if (entity is Hashtable hashtable)
+                                        ReadHashTable(r, hashtable);
+                                    else if (entity is Array array)
+                                        value = ReadArray(name, array, r);
+                                    else if (entity is IXmlSerializable xs)
```

Comment:
I added some coding guidlines, and one of them is that I prefer to keep curly braces on all if-statements, even simple ones like this. I would also prefer not to use the @ sign to protect identifiers. So just pick a different variable name from "string".